### ISSUE TITLE
Thunderbird 60.x unable to use caldav/carddav

### ISSUE THREAD

**Issue by egal88** [nextcloud/server#10134]
When trying to access calendar (Lightning) and addressbooks (CardBook) on a Nextcloud 13.0.4 server, Thunderbird 60.0b9 receives status 503 from the server and therefor cannot access any data.

Using Thunderbird 52.x everything works as expected.

Cross reference see https://bugzilla.mozilla.org/show_bug.cgi?id=1468912

The admin of our webserver tried the same with owncloud 7.0.4, but he did not test newer versions.
Using this really old version of owncloud, Thunderbird 60.0b9 manages to access the caldav and carddav urls without any error.
Since Nextcloud is a fork of owncloud, it might be, that the described issue is not Thunderbird related.
Perhaps the bug has to be searched for in Nextclouds code.

**Comment by painxcess** [comment]
I can confirm a working Thunderbird 60.x with ownCloud 10.0.2.
I can confirm a not working Thunderbird 60.x with Nextcloud 11.0.3 and nextcloud 12.0.9.

**Comment by MorrisJobke** [comment]
cc @georgehrke

**Comment by egal88** [comment]
Thunderbird 60.0b10 still has the same issue

**Comment by biva** [comment]
We might be investigating the same issue here: https://gitlab.com/CardBook/CardBook/issues/306 and https://bugzilla.mozilla.org/show_bug.cgi?id=1468755

Could you confirm that TB60 beta 4 did not have the issue, while beta 5 and following have it? (you can find these versions here: https://filehippo.com/fr/download_thunderbird/history/

**Comment by biva** [comment]
And for reference, I closed my initial issue because I thought it was cardbook related: https://github.com/nextcloud/server/issues/9869

@egal88 could you correct your bugzilla link (it's broken)?

**Comment by painxcess** [comment]
I can confirm:
- Beta 4 syncs caldav (works)
- Beta 5 does not sync (-> actual issue)

**Comment by egal88** [comment]
@biva Bugzilla link corrected. Thanks for your info.

**Comment by biva** [comment]
@georgehrke @MorrisJobke Thunderbird developers ask for Nextcloud developers' support in https://bugzilla.mozilla.org/show_bug.cgi?id=1468912#c33 as it might be specific to Nextcloud. Could you have a look?

**Comment by MorrisJobke** [comment]
@georgehrke Could you answer their questions, as you know a bit more about it.

**Comment by MorrisJobke** [comment]
Or maybe @rullzer regarding the auth stuff.

**Comment by biva** [comment]
@rullzer and @georgehrke It looks like if this issue happens with Nextcloud server only. Could you have a look at this Thunderbird issue? https://bugzilla.mozilla.org/show_bug.cgi?id=1468912#c54

**Comment by biva** [comment]
@adi-dev @georgehrke @rullzer Thunderbird 60 has been released and this issue is the single "Known Issues" in the release notes https://www.thunderbird.net/en-US/thunderbird/60.0/releasenotes/

And apparently, only Nextcloud is having this issue. Thunderbird developers are begging for a support from Nextcloud to investigate this (https://bugzilla.mozilla.org/show_bug.cgi?id=1468912#c61). Could you have a look?

**Comment by georgehrke** [comment]
When I took a look a couple of days ago it seemed to be related to Nextcloud in a  subdirectory. Can anyone confirm this? 

Looking into a proper fix now ..

**Comment by biva** [comment]
> it seemed to be related to Nextcloud in a subdirectory. Can anyone confirm this?

I'm not sure to understand your question. I'm having the issue and my Nextcloud is installed in `/var/www/html/nextcloud` and the url is https://nexcloud.mywebsite.com

**Comment by adi-dev** [comment]
Just out of curiosity, my httpd logs are full of:
```
503 Service Unavailable
    /remote.php/dav/calendars/username/contact_birthdays/: 8 Time(s)
	/remote.php/dav/calendars/username/pers ... 18671aca94f.ics: 8 Time(s)
	/remote.php/dav/calendars/username/pers ... 2B019A7E39A.ics: 8 Time(s)
	/remote.php/dav/calendars/username/pers ... 746adb10b4e.ics: 8 Time(s)
	/remote.php/dav/calendars/username/personal/: 8 Time(s)
	/remote.php/dav/calendars/username/work ... 3205226d8d2.ics: 8 Time(s)
	/remote.php/dav/calendars/username/work ... 424356a20b3.ics: 8 Time(s)
	/remote.php/dav/calendars/username/work ... 8f14bfddd52.ics: 8 Time(s)
	/remote.php/dav/calendars/username/work ... 98e7d1e3e30.ics: 8 Time(s)
	/remote.php/dav/calendars/username/work ... ac01859d6df.ics: 8 Time(s)
	/remote.php/dav/calendars/username/work ... afbc4852481.ics: 8 Time(s)
	/remote.php/dav/calendars/username/work ... d04937ec4a4.ics: 8 Time(s)
	/remote.php/dav/calendars/username/work/: 8 Time(s)
```
I set up my Thunderbird as CalDAV, is it normal it requests .ics files?
Any clues?

**Comment by j-ed** [comment]
@georgehrke I just read the description of the Thunderbird bug report and it seems that the interesting part starts at [comment 43](https://bugzilla.mozilla.org/show_bug.cgi?id=1468912#c43). By disabling the newly introduced parameter `network.cookie.same-site.enabled` the problem can be solved. So it seems to be a kind of cookie handling problem?!

**Comment by georgehrke** [comment]
> By disabling the newly introduced parameter network.cookie.same-site.enabled the problem can be solved. So it seems to be a kind of cookie handling problem?!

Yes, there is an issue with the cookie handling. I'm talking to the person at Mozilla who implemented it to see what causes this issue and how we can resolve it.

### As a workaround:
(as described in Thunderbird's [release notes](https://www.thunderbird.net/en-US/thunderbird/60.0/releasenotes/#known-issues)):
- go to the config editor (Settings -> Advanced -> General -> Config Editor)
- search for `network.cookie.same-site.enabled`
- set it to false

For your own security you should set it back to true once this issue is resolved.

**Comment by adi-dev** [comment]
I can confirm, the workaround works for me, cheers

**Comment by JSoko** [comment]
No problems with old installations upgraded to Thunderbird 60.

After creating a new profile in TB 60. had the problem with caldav and carddav under cardbook.
The network.cookie.same-site.enabled works, but I think, Thunderbird should work out of the box...

**Comment by goddib** [comment]
> When I took a look a couple of days ago it seemed to be related to Nextcloud in a subdirectory. Can anyone confirm this?

I can tentitavely confirm this (even though I assume debugging has moved beyond this point). I was actually going mad trying to figure out why one of my up-to-date nextcloud instances (in root folder) would sync as usual after the TB60 update and the other one wouldn't (in sub-folder, TB actually receives a 503). Especially as both kept synching with DAVDroid and react to curl.

The work around seems to work for me as well. Keep up the good work, thanks everyone!

**Comment by biva** [comment]
> Yes, there is an issue with the cookie handling. I'm talking to the person at Mozilla who implemented it to see what causes this issue and how we can resolve it.

Hello @georgehrke,
Did you find the causes?

**Comment by georgehrke** [comment]
> Did you find the causes?

No, we are still looking for what causes this. If we find out, I will comment here.

**Comment by MichaIng** [comment]
👍 Great both sides are working on this, this issue drove me crazy for the last two days 🤣.

**Comment by biva** [comment]
Still no idea about the cause? Thanks for your investigation.

**Comment by criwe** [comment]
I´m syncing two thunderbirds (both 60.0) on two computers with the same Nextcloud-Installation. Both same system (win7), but one is 32bit, the other is 64bit. The 32bit system seems to work fine (thunderbird syncs like expected), the 64bit doesnt want to sync (execpt using the workaround described above). May that helps...

**Comment by painxcess** [comment]
I can't confirm a working Windows 7 Pro in 32 Bit with Nextcloud 13.0.6.
(Tested a fresh Win7-Install 32 Bit with all updates and Thunderbird 60)

**Comment by mbiebl** [comment]
I'm also hit by this bug. 
What's interesting for me though, that I only have this problem with a fresh TB profile.
With an existing TB profile where I've setup lightning a long time ago, the calendar sync works.

**Comment by mnlipp** [comment]
@mbiebl cannot confirm the "only fresh profile" observation. Just tested two thunderbirds with year old profiles and had to activate the work-around for both.

**Comment by criwe** [comment]
@mnlipp: can confirm that old profiles got the problem too.

I´ve updated Nextcloud from 13.x to 14.01, but still the same issue...

**Comment by rullzer** [comment]
I'm not sure but https://github.com/nextcloud/server/pull/11433 might fix this.

**Comment by criwe** [comment]
@rullzer looks good. I´ve tested with two installations, in both syncing works fine now. 
But would be cool, if some other thunderbird user(s) can test too.

**Comment by biva** [comment]
I just installed NC 14.0.3: is this patch included?

**Comment by MichaIng** [comment]
@biva 
I don't think so, can't find the new `CookieHelper` class inside [stable14 branch](https://github.com/nextcloud/server/tree/stable14)

**Comment by criwe** [comment]
@biva @MichaIng I´m not sure, but I think so, because it #11433 is marked as merged and I´ve only made the update to  14.0.3 and no other manual patching.

**Comment by adi-dev** [comment]
Just tried, changed settings `network.cookie.same-site.enabled=true` and restart the TB and BOOM! my TB version just upgraded to 63 beta 1, none of the add-ons are working lol. Rolled back TB to 60.2.1 (release) and checked, all working with `network.cookie.same-site.enabled=true` 😃 🍻 cheers guys!

**Comment by fuzsin** [comment]
Neither Lightning nor CardBook works with Nextcloud 14.0.3 stable without network.cookie.same-site.enabled=**false** hack. TB 60.2.1 (32-Bit) fresh installed.

**Comment by blackcrack** [comment]
here works with Thunderbird 60.2.1 (32-Bit)
with the beta of sogo-connector-60.0.0-e2547a3565.xpi
=  https://packages.inverse.ca/SOGo/thunderbird/nightly/
calendar works
also CardBook works too 
and my NextCloud is 14.0.0.3
best regards
Blacky

![screen 00002](https://user-images.githubusercontent.com/7925717/46948919-eff47700-d07f-11e8-876b-524236889729.jpg)

**Comment by jadesoturi** [comment]
The workaround does not work for me.. NextCloud 13.0.7 with Thunderbird 60 x64 on Arch. As soon as i set the setting to FALSE, the calendar gets "disabled" in Lightning and i can reenable it again untill i change the config back to true..

**Comment by mnlipp** [comment]
For the sake of completeness: The problem still exists with Nextcloud 14.0.3 and TB 60.2.1 (64-Bit) (@fuzsin  already reported it for 32-bit). The workaround still works (and is still required).

**Comment by jadesoturi** [comment]
@mnlipp  - I've tried the workaround fix with both the the 32bit and 64bit on my 13.0.7 installation, but it does not work for me. Results in the calendar being turned off in Lightning, until I set the setting to TRUE again. Is there something I'm missing that can be the reason for this?

**Comment by rullzer** [comment]
> I just installed NC 14.0.3: is this patch included?

Nope this is only in master. So will be NC15

**Comment by abject** [comment]
I use TB60.2.1 (64bits) under Ubuntu 16.04.4 with a Nextcloud 14.0.3 freshly migraded.
The change of `network.cookie.same-site.enabled=true` to `network.cookie.same-site.enabled=false` allows the calendar synchronisation.

**Comment by blackcrack** [comment]
NC 14.0.3 on Openmandriva 3.03 Server .
Client TB 60.3.0 (32-Bit)
https://packages.inverse.ca/SOGo/thunderbird/nightly/
https://packages.inverse.ca/SOGo/thunderbird/nightly/sogo-integrator-60.0.0-3f871d5df1-sogo-demo.xpi
this works on TB WinNT
and also works the Carddav own solution of Thunderbird..
Open AdressBook => Extra => New => Remote-AdressBook 

best regards
Blacky

**Comment by aventu90** [comment]
> I use TB60.2.1 (64bits) under Ubuntu 16.04.4 with a Nextcloud 14.0.3 freshly migraded.
> The change of `network.cookie.same-site.enabled=true` to `network.cookie.same-site.enabled=false` allows the calendar synchronisation.

Thank you! This fixed the issue with Thunderbird 60.3.0. Calendar and Contacts are working now.

**Comment by qw3ry** [comment]
@jadesoturi the workaround worked for me (mostly fresh Arch installation, Nextcloud 14, don't know which minor version though)

**Comment by j-ed** [comment]
Unfortunately I ran into a lot of trouble as Thunderbird automatically updates to v60.3.0 (32bits) yesterday. Thunderbird wasn't able to show all my subscribed calendars nor was the response behavior acceptable with `network.cookie.same-site.enabled=true`, as it tries to synchronize with Nextcloud 14.0.3.
I managed to set `network.cookie.same-site.enabled=false` but the response behavior was still bad although some calendar items were shown. At the end I decided to reinstall Thunderbird 52.9.1 and got everything working again. Therefore it seems that I need to wait for a full fix in Nextcloud to get it working as before.

**Comment by MichaIng** [comment]
On a fresh Windows install, after `network.cookie.same-site.enabled=true` was somehow working for me, now it is not anymore, matching reported results here.

Perhaps when Thunderbird and Nextcloud already communicated successfully, cookies are saved and such, the setting can be set to `true` again? Will test by times.

**Comment by brunt82** [comment]
> When prompted for username and password I get the following warning:
my-hostname is requesting your username and password. WARNING: Your password will not be sent to the website you are currently visiting! What does this mean really? Could this have something to do with the issue?
No, this means that you retrieves your e-mails from another site than your calendar data.

**Comment by ftoral** [comment]
I'm also affected.
Workaround ok with `network.cookie.same-site.enabled=false`

**Nextcloud server version:** 13.0.1
**Thunderbird:** 60.3.0
**Lightning:** 6.2b6

**Comment by georgehrke** [comment]
No further comments that it doesn’t work on Nc 13/14 with Thunderbird 60 please. We are aware and more and more comments won’t fix it ;):) 

If you want to help us fix this, please grab the RC of Nextcloud 15 and verify that it fixes it.

**Comment by fuzsin** [comment]
NextCloud 15 beta **RC3**, Thunderbird 60.3.2 (32-Bit), Lightning 6.2.3.2, Windows 10 (64-Bit) - same issue, workaround works.

**Comment by Schroedingers-Cat** [comment]
@Luegengladiator if you fixed the issue by removing the calendars and reimporting them. how do you end up with the conclusion that this is a DNS problem? I don't see any relation there.

@georgehrke as there are already two users reporting the issue still exists, is there anything we can do to help so this issue will get fixed?

**Comment by Luegengladiator** [comment]
@Schroedingers-Cat 
I removed the calendar referencing http://my.local.domain/path/to/calendar  (nonworking)
With http://127.0.0.1/path/to/calendar (working)
So the only thing I changed was replace a Name with the corresponding IP. 
I think this is a local DNS Problem ;)

**Comment by mbiebl** [comment]
Tested with version 15.0.0 and Thunderbird 60.2.1 and 60.3.1 and was hoping this would be fixed.
Problem is unfortunately *not* fixed so #11433 did not help unfortunately.

Would really be great to have some focus on this issue. This affects a lot of users and is very simple to reproduce.

**Comment by georgehrke** [comment]
> Would really be great to have some focus on this issue. This affects a lot of users and is very simple to reproduce.

There is focus on this issue. You, being a developer yourself, should know best that simple to reproduce is not the same as simple to debug / simple to fix 😉 

Having the same logic for same-site cookies work with all major browsers and other Calendar applications just fine, I tend to say this is a bug in Thunderbird / Lightning, but we are still figuring that out together with the people at Mozilla who wrote the Same Site Cookie code.

**Comment by lyu0001** [comment]
Solved the problem via option --> General --> config editor --> network.cookie.same-site.enabled=false
Nextcloud server version: 15.0.0
Thunderbird: 60.4.0
Lightning: 6.24

**Comment by rvanlaar** [comment]
> I have found a safer workaround: If you have changed it already set network.cookie.same-site.enabled to true again. Then set all third party cookies to be denied and add an exception for the domain your Nextcloud installation is running on.

@svenmauch Could you walk me through the steps to add an exception for nextcloud?

**Comment by MichaIng** [comment]
@rvanlaar 
Settings > Privacy > Web content > Exceptions > Enter `https://your.domain.org`[`/nextcloud`]

**Comment by mnlipp** [comment]
> I have found a safer workaround: If you have changed it already set network.cookie.same-site.enabled to true again. Then set all third party cookies to be denied and add an exception for the domain your Nextcloud installation is running on.

Does not work for me (60.3.1).

**Comment by Schroedingers-Cat** [comment]
The workaround by @svenmauch and @mjmunger does not work for me (TB 60.4, Nextcloud 13.0.8).

**Comment by georgehrke** [comment]
I created a topic for workarounds in our forum: https://help.nextcloud.com/t/workarounds-for-same-site-cookie-bug-with-thunderbird/44873

As the developer, I recommend to stick to the workaround described in https://github.com/nextcloud/server/issues/10134#issuecomment-411322692

Please take all further discussion about workarounds to that forum topic.

**Comment by mbiebl** [comment]
Is there a way to disable this functionality on the server side? I'm asking because this particular issue is what's currently preventing me from upgrading one of my nextclound instance to v15.
This installation has quite a few users and a lot of them do use TB. Asking all those users to update/change their TB configuration is not really an option.
If I could change the cookie handling on the server side and switch it back once fixed, that would be workable solution for me.

**Comment by HorlogeSkynet** [comment]
I can confirm that this is happening on the v14 too.
It may be something related to the Thunderbird configuration, accounting for the second workaround not working as expected for everyone (?).

@mbiebl It is definitely not recommended to tweak cookie settings on the server-side, as it would endanger ALL your clients, including regular WEB usage, and not only Thunderbird synchronization.

**Comment by gohrner** [comment]
> Is there a way to disable this functionality on the server side? I'm asking because this particular issue is what's currently preventing me from upgrading one of my nextclound instance to v15.

Not upgrading NextCloud won't help you - at least V13 and V14 are affected as well.

You will have to *keep your users from updating Thunderbird* to V60, which in turn probably would be a bad idea from the security perspective. (Not even speaking of the difficulty of preventing users to update their software which possibly also auto-updates.)

So giving your users instructions of how to configure Thunderbird to work around the issue probably is the best approach.

**Comment by j-ed** [comment]
I had trouble with this function too in the past with Thunderbird v60.3.1, but after I exactly followed the tip of @svenmauch, I was successfully able to configure Thunderbird 60.4 with Lightning 6.2.4/Cardbook 33.5 and NC 14.0.4/.5.
I would recommend to always give it a personal try before relying on others feedback. There are so many system in the wild, which are all configured differently, that you couldn't rely on it.

**Comment by tleydxdy** [comment]
the [safer workaround](https://github.com/nextcloud/server/issues/10134#issuecomment-450732634) didn't work for my setup (Thunderbird 60.4.0 arch, and Nextcloud 15.0.2)

**Comment by JtheBAB** [comment]
Thundebird found the issue:

https://bugzilla.mozilla.org/show_bug.cgi?id=1468912

See latest comments.

**Comment by rfc2822** [comment]
Ok thanks… I got it now working but I have to delete the credentials/cookies every time I want to see the other calendar. Confusing but it somehow works now.

**Comment by georgehrke** [comment]
@rfc2822 Hey, the recommended workaround is the one i mentioned [further above](https://github.com/nextcloud/server/issues/10134#issuecomment-411322692).

> Has this been fixed already (either in Thunderbird or in NC or in both)?

Looking at the latest comments in https://bugzilla.mozilla.org/show_bug.cgi?id=1468912, the issue was fixed on the Thunderbird side and I expect a fix to be released soon.

**Comment by georgehrke** [comment]
I'm closing this ticket, because it's an issue in Thunderbird. 

Looking at https://bugzilla.mozilla.org/show_bug.cgi?id=1468912#c83, they already have a fix and consider releasing a new beta version of Lightning 6.7, so i expect this fix to be available soon.

**Comment by darmbrust** [comment]
Has anyone actually confirmed that things got fixed?  It still doesn't work for me today,  using thunderbird 60.7.0, CalDav 0.15.0.2, TbSync 1.7 against NextCloud 16.0.3.  The TbSync folks seems to have closed it, just telling people to use the security hack workaround... https://github.com/jobisoft/DAV-4-TbSync/issues/41  but its really hard to track which bugs are actually tracking this, as so many were opened across different projects, then closed again, claiming fixed... yet it still doesn't work.

**Comment by MichaIng** [comment]
@darmbrust 
Why not just test it?
I just did, switched `network.cookie.same-site.enabled` back to true (default), closed Thunderbird, removed all cookies via CCleaner (shame on me...), reopened Thunderbird, toggled and re-synced my Lightning calendars and CardBook and it works 😃.
I also created a new calendar and successfully synced with Lightning.
So as long as there is no cache influence or something I forgot, the issue indeed seems to be resolved (Lightning 6.2.8).

**Comment by blackcrack** [comment]
you should read the manual of the "Videorecorder" ! \*veg* (Very Evel Grin)

**Comment by darmbrust** [comment]
Well, I _did_ test it, thats how I got here.  I recently set up nextcloud... installed the calendar app today, and tried to sync it with thunderbird, and could not successfully link to my calendar.  Not until I changed the cookie setting.  And this was with a clean install that has never attempted this before.  Happy to change it back and get debug logs... if there is something you want to see....

**Comment by georgehrke** [comment]
@darmbrust I can confirm that plain Thunderbird Lightning without TbSync works.

As pointed out in the [mozilla bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1468912#c83), it was a bug in the mozilla implementation. So if it's still not working in DAV-4-TbSync, perhaps open a new bug report there?

**Comment by darmbrust** [comment]
well, as recently as 15 days ago, people are commenting on that bug that it isn't fixed either.

Though, perhaps they are using the DAV plugin too.

**Comment by darmbrust** [comment]
So, I upgraded Thunderbird to 60.8.0.  Tried adding my nextcloud calendar via the Network -> CalDev option directly in Lightening.  It adds the calendar... but shows nothing.  Sync does nothing.

No errors are shown to the user.  

Error log contains stuff like:
503 Service Unavailable.
[calCachedCalendar] replay action failed: null,
uncaught exception.

Toggled network.cookie.same-site.enabled to false, and suddenly the calendar syncs.

So, yea.... probably on the thunderbird side, but still not fixed... and doesn't work at all without finding this obscure toggle.

Of course, Lightening also doesn't let you view or edit the events properly either... they all come up with garbage times if you try to view them... but guessing that is on the lightening side too, since they show up on the calendar GUI with the correct dates and times.

**Comment by brunt82** [comment]
I cannot confirm any of your described behavior: Neither with Lightning nor with TBSync. 
I tested it with: 
- TB 60.8, TBSync 1.7, CalDavProvider 0.15.0.2, Nextcloud 15.0.10
- TB 60.8, Lightning 6.2.5.1, Nextcloud 15.0.10
"network.cookie.same-site.enabled" is set to true in both cases.

Maybe it is any other issue?

**Comment by NilsDacke** [comment]
Works for me:

- TB 60.8, CardBook 37.5, Nextcloud 15.0.10
- TB 60.8, Lightning 6.2.8, Nextcloud 15.0.10

network.cookie.same-site.enabled = true

**Comment by darmbrust** [comment]
Maybe this is related to the issue, but I don't understand how people keep reporting they are testing with Lightning 6.2.8 when 6.2.5 is the newest version I see:
https://addons.thunderbird.net/en-US/thunderbird/addon/lightning/versions/

The other difference in versions with my setup, compared to the recent works for me posts is that I'm on NextCloud 16.0.3.

Happy to grab logs / debug from whatever parts people would like to see.

**Comment by darmbrust** [comment]
Ok, guessing this is where the fix landed:
https://bugzilla.mozilla.org/buglist.cgi?resolution=FIXED&product=Calendar&target_milestone=6.2.5.1&list_id=14828679

6.2.5.1, while my calendar is still at 6.2.5. According to this list:
https://developer.mozilla.org/en-US/docs/Mozilla/Calendar/Calendar_Versions
I'm like 5 versions behind on Lightning.... I know its not your issue, but what the heck is wrong with the Lightning distribution that they aren't pushing their updates to the central update site?

Sigh... reading more, it appears that they want you to uninstall the extension, cause its no longer necessary... as its supposed to be bundled. That is the stupidest update path I've ever heard of.... something tells me they will bite a lot of people with this, when they didn't manage the extension properly for users that already had it installed.... and they aren't properly updating the update site, so if you installed it from there, the older version overrides the integrated version.

I'll retest after sorting this out so I actually get the current version of lightning.  If anyone else encounters this bug, it appears that the solution is to uninstall the lightning extension, so that thunderbird will go back to using the embedded one released with it, and not the out-of-date broken version mozilla has published on the update site.

**Comment by MichaIng** [comment]
@darmbrust 
What is wrong about the TB internal addons management? I installed Lightning from there and are automatically updated to the latest version since long time.

Lol this addons web page gets even worse when you download from the main Lightning page: https://addons.thunderbird.net/en-US/thunderbird/addon/lightning/
You will get 6.2(.0) there 😄. However as said I see no reason to use this website for downloading+installing addons manually anyway, at least in case of a single home PC.

**Comment by darmbrust** [comment]
Ok, there is just a whole lot of stupid with the Mozilla / Lightning / Ubuntu package maintenance folks.  I installed Thunderbird using the package manager, but they stripped Lightning, which is supposed to be included.  Lightning as published by mozilla is 5 versions out of date.  If you uninstall the extension again, its just gone, because Ubuntu stripped it.

To get the proper, up-to-date version, you have to install it using the Ubuntu package manager, (assuming you let ubuntu install thunderbird in the first place) which restores the extension they stripped, and you get the proper 6.2.8 version.

After sorting that out, I can confirm that it is indeed fixed, when using the native remote caldav option.  Though I am still seeing issues when trying to use the TBSync addon.  But guessing that problem is over there... https://github.com/jobisoft/DAV-4-TbSync/issues/100

**Comment by MichaIng** [comment]
Okay, then finally things seem to match. Perhaps for @brunt82 the same works, as you're on v6.2.5 as well.

**Comment by r3pek** [comment]
Anyone here got it to work with thunderbird 68.x? I can't even add an account on TbSync :(

**Comment by MichaIng** [comment]
@r3pek 
I suggest you open a new issue for this to not mix things with the different 60.x bug that was discussed+solved here.

**Comment by j-ed** [comment]
@r3pek I would recommend to ask for help in the [official Nextcloud help forum](https://help.nextcloud.com) first, because this is repository is primarily used to track bugs 😉

**Comment by blackcrack** [comment]
@r3pek , use the f..\*beep* 68.x and lost the whole nice plugin.. never.. the thunferbird developing goes not the way what's the plugins supports.., they should schame what they are made in the name of "safety&security" bahh.. i hold my 60.x and there works all plugins.. so far what's is usable ..

best regards
Blacky

p.s.: "but the savety and datasucurity" Bullshit, it can also overdo it ! The thunderbird is comes up because the possible of plugins !

**Comment by MichaIng** [comment]
I just realized that TB 68 is now released and not beta anymore. However it is common that not all plugins support new versions immediately.

Not sure about the Nextcloud plugin (Lightning/CardBook), will try as well later, but this should then discussed in a separate issue or indeed Nextcloud forum first to verify and collect some info.

**Comment by xpressio** [comment]
I still cannot get syncing to work in our lab (5+ users). We are on Nextcloud 14/15 using TB 60.x or TB 68.x. Can anyone confirm that upgrading to NC 16 definitively solved their issues?

In my experience the recommended fix using `network.cookie.same-site.enabled` ceased to work some months ago. It used to work in earlier versions of TB, I have tested various combinations of NC, TB and Lightning - to no avail...

Any experiences / suggestions would be much appreciated - using the web interface for now but usability is less then ideal.

**Comment by j-ed** [comment]
> Any experiences / suggestions would be much appreciated - using the web interface for now but usability is less then ideal.

Unfortunately this is the bug tracker for the development and **NOT the** [**Nextcloud help forum**](https://help.nextcloud.com/). You should use the search function of the help forum to find an answer on your question first, before you raise a new request.
I can personally confirm that the original problem described here has already been solved and Thunderbird 68.2.2/Cardbook 43.0 work like a charm with Nextcloud 17.0.1.

**Comment by manoranjith** [comment]
I was still facing sync issues with Nextcloud 19 and Thunderbird 68.10 (linux). 

SOLVED: I was able to make it work using the [TbSync](https://addons.thunderbird.net/en-US/thunderbird/addon/tbsync/) and [Provider for CalDAV & CardDAV](https://addons.thunderbird.net/en-US/thunderbird/addon/dav-4-tbsync) plugins as suggested in [this thread](https://help.nextcloud.com/t/solved-thunderbird-caldav-sync-in-linux-not-working-with-nextcloudpi/61393/2) in nextcloud forum.
